packages/api-explorer/src/Params.jsx

```diff
@@ -128,8 +128,8 @@ class Params extends React.Component {
           this.jsonSchema.map(schema => {
             return (
               <React.Fragment key={`${schema.type}-block`}>
-                {enableJsonEditor ? (
-                  <React.Fragment>
+                {enableJsonEditor && schema.type === 'body' ? (
+                  <div className="param-type-body">
```

Comment:
This change here is a small risk to breaking any custom CSS if a customer has a specific selector or prop that only affects direct child node (ie. `.param-type -> .param-type-header`, or `.param-type {display: flex}`)

Reply:
Should be fine since we just have Envoy using this stuff right now

Reply:
Oh I see. Will test!

Reply:
@runnabro Keeping both `position: relative` rules around it seems okay?

![Screen Shot 2020-12-17 at 8 53 39 PM](https://user-images.githubusercontent.com/33762/102576277-fd677a80-40a9-11eb-8143-544e4c872b66.png)

Reply:
Oh, we don't actually need both, ill remove the parent one, thanks!